File: algoritmos-e-logica-de-programacao/PyCharm/ws-pycharm/soma-impares.py
somaImpares = 0

print("Digite dois numeros:")
x = int(input())
y = int(input())

if x > y:
    troca = x
    x = y
    y = troca

for i in range(x + 1, y):
    if i % 2 != 0:
        somaImpares = somaImpares + i

print()
print("SOMA DOS IMPARES = ", somaImpares)

File: algoritmos-e-logica-de-programacao/Eclipse/ws-eclipse/curso_programacao/src/SomaImpares.java
import java.util.Locale;
import java.util.Scanner;

public class SomaImpares {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		
		Scanner sc = new Scanner(System.in);
		
		int x, y, troca, somaImpares = 0;
		
		System.out.println("Digite dois numeros:");
		x = sc.nextInt();
		y = sc.nextInt();
		
		if (x > y) {
			troca = x;
			x = y;
			y = troca;
		}
		
		for (int i = x + 1; i < y; i++) {
			if (i % 2 != 0) {
				somaImpares = somaImpares + i;
			}
		}
		
		System.out.println();
		System.out.println("SOMA DOS IMPARES = " + somaImpares);
		
		sc.close();
	}

}
